data/os/Suse.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-bacula_director_packages: [ 'bacula-director-common', "bacula-director-${db_type}", 'bacula-console' ]
+bacula_director_packages: [ 'bacula-director-common', "bacula-director-<%= $db_type %>", 'bacula-console' ]
```

Comment:
Odd.  I don't actually see this uesd currently.

Reply:
Yes, it bugged me a bit… Some old Debian release have a **bacula-sd-*database*** package containing utilities like `bscan` used to analyse backups when you lost your database:
https://packages.debian.org/jessie/bacula-sd-pgsql

This package is an old one, available on Debian oldoldstable and sid (not updated).  I never used any of these packages, and I assumed this code existed because it was required for legacy Debain releases I have not been using with the module.  I guess that when we drop oldoldstable support, we can simplify this…